src/hooks/useHover.ts

```diff
@@ -16,26 +16,39 @@ limitations under the License.
 
 import React, { useEffect, useState } from "react";
 
-export default function useHover(ref: React.MutableRefObject<HTMLElement>) {
+export default function useHover(ref: React.MutableRefObject<HTMLElement>, ignoreAnchors = false) {
```

Comment:
I don't think we should have a very specific prop like this in such a generic hook.
If we want to have a escape hatch to ignore certain element then fair enough, but let's make devs pass a function where they can handle the logic outside of this hook. Otherwise we will have an issue when we inevitably want to ignore paragraphs or any other tags